ush/remap.sh

```diff
@@ -0,0 +1,376 @@
+    exit 100
+fi
+
+#######
+
+# _comma_split_string - Split a comma-delimited string into an array.
+#
+# Description:
+#   This function takes a comma-delimited string as input and splits
+#   it into an array. Each element in the resulting array is obtained
+#   by splitting the input string at commas and then removing leading
+#   and trailing spaces.
+#
+# Parameters:
+#   $1 - The comma-delimited string to split.
+#
+# Global Variables:
+#   global_array - An array containing the split elements.
+#
+# Example usage:
+#   _comma_split_string "item1,item2 item3,item4"
+#   for element in "${global_array[@]}"; do
+#       echo "$element"
+#   done
+#
+# This example will split the input string into individual elements
+# and print each element on a separate line.
+function _comma_split_string() {
+    local string="${1}"
+
+    local local_array=()
+    global_array=()
+    IFS="," read -ra items <<< "${string}"
+    for item in "${items[@]}"; do
+        local_array+=("${item} ")
+    done
+    for item in "${local_array[@]}"; do
+        IFS=" " read -ra items <<< "${item}"
+        for element in "${items[@]}"; do
+            global_array+=("${element} ")
+        done
+    done
+}
+
+#######
+
+# _strip_whitespace - Remove whitespace from a string.
+#
+# Description:
+#   This function takes an input string and removes all whitespace
+#   characters (spaces, tabs, and newline characters) to produce a
+#   cleaned output string.
+#
+# Parameters:
+#   $1 - The input string from which whitespace will be removed.
+#
+# Return:
+#   The cleaned string with no whitespace.
+#
+# Example usage:
+#   cleaned_string=$(_strip_whitespace "   This is a string   with spaces  ")
+#   echo "Cleaned string: \"$cleaned_string\""
+#
+# This example will remove all leading, trailing, and internal
+# whitespace from the input string and display the cleaned result.
+function _strip_whitespace(){
+    local in_string="${1}"
+
+    out_string=$(echo "${in_string}" | sed "s/ //g")
+}
+
+#######
+
+# nc_concat - Concatenate netCDF files and manage file paths.
+#
+# Description:
+#   This function performs the concatenation of two netCDF files:
+#   `output_path` and `var_interp_path`. If the `output_path` file
+#   exists, the function merges `var_interp_path` into it and manages
+#   temporary files accordingly. If `output_path` does not exist, it
+#   creates a new netCDF file by renaming `var_interp_path` to
+#   `output_path`.
+#
+# Parameters:
+#   $1 - The path to the variable-interpolated netCDF file.
+#
+# Global Variables:
+#   output_path - The path to the output netCDF file.
+#
+# Example usage:
+#   nc_concat "variable_interpolated.nc"
+#
+# This example will concatenate the variable-interpolated file into
+# the output netCDF file, handling the case where the output file may
+# or may not exist.
+function nc_concat(){
+    local var_interp_path="${1}"
+
+    tmp_nc_file="${PWD}/tmp_nc.nc"
+    if test -e "${output_path}"; then
```

Comment:
```suggestion
    if [[ -e "${output_path}" ]]; then
```

Reply:
The contents of this if-block is also not indented properly.

Reply:
Can you point me to the correct way to indent these blocks?

Reply:
It looks like this on my screen (the block is flush with the surrounding code):
![image](https://github.com/NOAA-EMC/global-workflow/assets/28147896/e07ad553-a1c9-4e71-bfb2-b98ec8e79f01)

Make sure you are using only spaces for indentation. It looks like this on GH sometimes if tabs are mixed in.

Reply:
Done, at least it appeared correctly in my GH terminal.